pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -696,3 +706,63 @@ func maybeUpdateConfigItems(ctx *volumemgrContext, newConfigItemValueMap *types.
 		}
 	}
 }
+
+//returns empty if not found
+func getZVolsByName(ctx *volumemgrContext, poolAndName string) string {
```

Comment:
It feels like these helper methods all belong in the zfs package

Reply:
ByName or ByDataset as you have below? 
The comment explaining the key and value for the map is missing; should presumably be added where the map is declared.

Reply:
I remove this kind of methods worked with map in favor of using mdev to create symlinks in /dev/zvols